notify-debouncer-full/Cargo.toml

```diff
@@ -19,12 +19,14 @@ name = "notify_debouncer_full"
 path = "src/lib.rs"
 
 [features]
-default = ["crossbeam"]
+default = ["crossbeam","macos_fsevent"]
 # can't use dep:crossbeam-channel and feature name crossbeam-channel below rust 1.60
 crossbeam = ["crossbeam-channel","notify/crossbeam-channel"]
+macos_fsevent = ["notify/macos_fsevent"]
```

Comment:
Is there a reason for masquerading the notify features? We simply let user specify them in their own cargo toml until now.